src/main/java/org/apache/mesos/executor/CustomExecutor.java

```diff
@@ -111,6 +115,45 @@ public void launchTask(final ExecutorDriver driver, final Protos.TaskInfo task)
         }
     }
 
+    private void executeHealthCheck(Protos.TaskInfo taskInfo, ExecutorTask executorTask) {
```

Comment:
maybe call this 'scheduleHealthCheck'? looks like it's scheduling a regular retriggering of the check, whereas 'execute' implies it's only being run once